examples/dyanamically-generated-background.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Trianglify - Dynamically Generated Background</title>
+
+    <style>
+      div#background {
+        width: 600px;
+        height: 120px;
+        background-size: cover;
+        background-repeat: no-repeat;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="background">
+      Let's fill this div with text adn see if that changes anything
+    </div>
+  </body>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/trianglify/3.0.0/trianglify.min.js"></script>
+
+  <script>
+    // Create Trianglify pattern feel free to use whatever settings here
+    // **NOTE: Must export as svg with namespace included ".svg({ includeNamespace: true })"
+    var pattern = Trianglify({
+      height: 100,
+      width: 300,
+      cell_size: 10,
+    }).svg({ includeNamespace: true });
+
+    // Take Trianglify SVG pattern and serialize it into XML string
+    var patternString = new XMLSerializer().serializeToString(pattern);
+
+    // URL encode the pattern and set into the proper format for SVG background
+    var patternMin = 'url("data:image/svg+xml,' + patternString.replace(/"/g, "'").replace(/#/g, "%23").replace(/</g, "%3C").replace(/>/g, "%3E") + '")';
```

Comment:
When escaping content for data uris, best practice is to us [encodeURIComponent](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) rather than a set of regexes.

Reply:
Wasn't familiar with that, thanks for letting me know!